admin/partials/admin-monk-translation-list-render.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Provide the view for the monk_translation_list_render function
+ *
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Admin/Partials
+ */
+
+	$active_languages = get_option( 'monk_active_languages' );
+	$suported_languages = array(
+		'pt_BR' => __( 'Portuguese (Brazil)', 'monk' ),
+		'en_US' => __( 'English', 'monk' ),
+		'es_ES' => __( 'Spanish', 'monk' ),
+		'fr_FR' => __( 'French', 'monk' ),
+		'it_IT' => __('Italian', 'monk')
+	);
+	$default_language = get_option( 'monk_default_language' );
+?>
+	<fieldset>
+	<?php
+		foreach ($suported_languages as $lang_code => $lang_name) {
+
+			/**
+			 * Walks through each value in the active languages array creating the checkboxes
+			 * based on the values, making possible and more reliable updating that list 
+			*/
+
+			$id = strtolower( str_replace( '_', '', $lang_code) );
+	?>
+			<label for="monk-<?php echo $id; ?>">
```

Comment:
Use checkboxes in WordPress style (copy the HTML structure):

![screen shot 2016-10-02 at 10 28 21 pm](https://cloud.githubusercontent.com/assets/5307192/19025470/95c71bc2-88ef-11e6-9f4e-419b6b56a5b4.png)
_print taked on General > Discussion menu_